README.rst

```diff
@@ -140,3 +141,9 @@ Options
       -a ATTACHMENT_SIZE, --attachment-size ATTACHMENT_SIZE
                             Attachment size limit in bytes. Set to -1 to disable
                             the limit
+
+Backup mode
```

Comment:
The script already has the word "backup" in the name. Should this feature be renamed to "incremental" instead?